helix-rest/src/test/java/org/apache/helix/rest/server/TestResourceAccessor.java

```diff
@@ -75,8 +75,17 @@ public void testGetResources() throws IOException {
 
     Set<String> resources = OBJECT_MAPPER.readValue(idealStates,
         OBJECT_MAPPER.getTypeFactory().constructCollectionType(Set.class, String.class));
-    Assert.assertEquals(resources, _resourcesMap.get("TestCluster_0"), "Resources from response: "
-        + resources + " vs clusters actually: " + _resourcesMap.get("TestCluster_0"));
+    Assert.assertTrue(resources.size() == _resourcesMap.get("TestCluster_0").size()
+        && resources.containsAll(_resourcesMap.get("TestCluster_0"))
+        && _resourcesMap.get("TestCluster_0").containsAll(resources),
+        "Sets are not equal. _resourcesMap.size(): "
```

Comment:
The original error message is `"Resources from response: " + resources + " vs clusters actually: " + _resourcesMap.get("TestCluster_0")`
It is better no to change it and in my opinion the long error message is too long to make any sense